jooq/src/main/java/io/micronaut/configuration/jooq/graal/SimpleFlatMapperAvailable.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2017-2020 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.configuration.jooq.graal;
+
+import java.util.function.BooleanSupplier;
+
+/**
+ * Companion class for DefaultReflectionServiceSubstitute.
+ *
+ * @author Lukas Moravec
+ * @since 2.3.5
+ */
+public class SimpleFlatMapperAvailable implements BooleanSupplier {
```

Comment:
Does this need to be public? If so please annotate with `@Internal`

Reply:
@ilopmar tested this and he wrote this on previous review:

> And the condition (the class needs to be public)

I will add the internal annotation, thank you.

Reply:
Gotcha.. thanks